v3/src/models/data/collection.ts

```diff
@@ -181,8 +179,8 @@ export const CollectionModel = V2Model
     if (!self.prevCaseIds) self.prevCaseIds = self.caseIds
     self.caseIds = []
 
-    if (!self.prevCaseIdToIndexMap) self.prevCaseIdToIndexMap = self.caseIdToIndexMap
-    self.caseIdToIndexMap = new Map<string, number>()
+    // no need to maintain a prev copy
```

Comment:
Seems like this comment is unnecessary?

Reply:
At the time I wrote it I thought people might wonder why all of the other properties needed to have prev copies maintained but not this one, but if you don't think it's necessary I'm happy to remove it.